api/app/src/command/medical/patient/get-patient.ts

```diff
@@ -19,6 +21,39 @@ export const getPatients = async ({
   return patients;
 };
 
+export const getPatientByDemo = async ({
+  facilityId,
+  cxId,
+  demo,
+}: {
+  facilityId: string;
+  cxId: string;
+  demo: PatientData;
+}): Promise<Patient | null> => {
```

Comment:
NABD, but IIRC we usually have `undefined` as not present

Reply:
For findOne it returns a patient or null

Reply:
Yeah, that's Sequelize's return, but we can convert to `undefined`.

NABD to leave as is if you want, I'll bring it up on a tech talk 👍